jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -349,19 +349,23 @@ def test_population_count(self, harness: primitive_harness.Harness):
 
   @primitive_harness.parameterized(primitive_harness.lax_binary_elementwise)
   def test_binary_elementwise(self, harness):
-    if harness.params["dtype"] is dtypes.bfloat16:
-      raise unittest.SkipTest("bfloat16 not implemented")
-    if harness.params["lax_name"] in ("igamma", "igammac", "rem", "atan2"):
-      # b/158006398: TF kernels are missing for 'rem' and 'atan2'.
+    if harness.params["lax_name"] in ("rem", "atan2"):
+      # b/158006398: TF kernels are missing for 'rem' and 'atan2'
+      if harness.params["dtype"] in [np.float16, dtypes.bfloat16]:
+        raise unittest.SkipTest("TODO: TF kernels are missing for 'rem' and 'atan2'.")
+      # TODO(bchetioui): do we need that here too?
+      if harness.params["dtype"] is np.float32 and jtu.device_under_test() == "tpu":
+        raise unittest.SkipTest("TODO: fix bug: nan vs not-nan")
+    if harness.params["lax_name"] in ("igamma", "igammac"):
       # TODO(necula): fix bug with igamma/f16
```

Comment:
I guess we can remove this TODO if not a bug in jax2tf

Reply:
The question is more about the fact that that https://github.com/google/jax/pull/4010 just added `rem`/`atan2` to the list of problematic operations for float16/bfloat16, but this also encompassed this condition for which it is not directly obvious that it should also apply (at least to me).
What do you think?

Reply:
After our chat offline, I removed this TODO.